Entries/copy-X/copy-X.md

```diff
@@ -168,36 +186,187 @@ No samples currently known, and thus no information either.
 #### Confirmed:
 
 * copy-X light
-	* Black Cats and Pointed Hats (USA) (En,De)[^84759]
+	* Black Cats and Pointed Hats (USA) (En,De) [^84759]
 	* Robinson Crusoe (USA)[^107929]
 	* Webmaster (USA)[^81628]
 	* Kenny's Adventure[^Kenny]
 	* Max and the Magician (Europe) (En,Fr,De,Es)[^101786]
+	* Die CD-ROM mit der Maus 3 (Germany) (EAN 9783898870030) [^Maus3_txt][^Maus3_IGDB]
+	* Meister Zufall und die Herrscher der Elemente (Germany) (Rerelease) (2000) (EAN 9783931372514)[^Zufall_ring_txt]
+	* Webmaster (Germany)[^116418][^Webmaster_txt]
+	* Alberts mysterioese Insel (Germany)[^116827][^Insel_txt]
+	* Onkel Alberts geheimnisvolles Notizbuch (Germany)[^116828][^Albert_txt]
+	* Westerner 2, The (Germany) [^DE_Westerner_2][^pcgames_de_westerner2][^gamestar_de_westerner2]
 * copy-X professional[^82475]
 	* Case for TKKG, A - The Mystery of the Mayan Treasure (USA) (En,De)[^108150]
 	* Emergency 4 - Global Fighters for Life (Germany) (Disc 1)[^48393]
 	* German Emergency 2 Deluxe[^DE_EM2DX]
 	* Petterson und Findus - Hühnertanz und Katerzirkus[^Petterson]
 	* Sluzhba 911 (Russia)[^82475] - Remnants (Official Russian release of Emergency 2 with protection removed)
+	* Die CD-ROM mit der Maus 1 (Germany) (Rerelease) (EAN 9783931372965) [^Maus_txt][^Maus_IGDB]
+	* Die CD-ROM mit der Maus 2 (Germany) (EAN 9783934789067) [^Maus2_txt][^Maus2_IGDB]
+	* Genius: Task Force Biologie (Germany)[^Genius_Physik]
+	* Genius: Unternehmen Physik (Germany)[^Genius_Biologie]
 
 #### Unconfirmed: 
 
 * copy-X (Unsure which variant):
-	* Alberts mysterioese Insel (Germany)[^116827]
 	* Ein Fall fuer TKKG - Das geheimnisvolle Testament (Germany)[^108587]
-	* Onkel Alberts geheimnisvolles Notizbuch (Germany)[^116828]
-	* Webmaster (Germany)[^116418]
-* copy-X light:
-	* German Westerner 2[^pcgames_de_westerner2][^gamestar_de_westerner2]
 * copy-X audio:
 	* Die Wilden Kerle – DWK4 - Der Angriff Der Silberlichten (Das Hörspiel Zum Kinofilm)[^D_8232013]
 	* Laava – Wherever You Are (I Feel Love)[^D_1436646]
 	* Manfred Mann* – 2006[^D_2182905]
 	* De-Phazz – Godsdog[^D_451419]
 
+Companies:
```

Comment:
Please add a new section entirely for "Companies Known to Use", similar to "Known Manufacturing Information". This section is intended to focus on confirmation of the DRM itself, not who used it.

Reply:
Tried doing this. Let me know what you think of it.